core/src/integration-test/java/com/scalar/db/storage/cassandra/ConsensusCommitCrossPartitionScanIntegrationTestWithCassandra.java

```diff
@@ -13,7 +13,7 @@ public class ConsensusCommitCrossPartitionScanIntegrationTestWithCassandra
 
   @Override
   protected Properties getProps(String testName) {
-    Properties properties = CassandraEnv.getProperties(testName);
+    Properties properties = ConsensusCommitCassandraEnv.getProperties(testName);
```

Comment:
These integration test classes need to use `ConsensusCommitCassandraEnv` to enable the group commit feature